website/docs/components/dropdown/partials/code/how-to-use.md

```diff
@@ -205,6 +214,15 @@ If you pass an `@href` argument, a link (`<a>` element) will be generated:
 </Doc::ListContainer>
 ```
 
+You can pass a `@trailingIcon` argument to indicate that the link points to an external resource:
```

Comment:
I know the example should be enough for people to understand but, to me, this paragraph reads as if the `@trailingIcon` sets an external link automatically.

Maybe something along the lines of:
To indicate that the link points to an external resource you can set `@trailingIcon` to `external-link`.

Reply:
@alex-ju the point is that the icon is not necessarily `external-link` in theory can be any icon.

Reply:
I see what you mean – but not _any_ icon (say `rocket`) _will_ indicate that the link points to an external resource. let's leave it as is then and if we get any feedback we can think of how to rephrase it.

Reply:
I've updated the paragraph, see if it sounds better.